Gordon360/Models/ViewModels/RecIM/MatchTeamViewModel.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Gordon360.Models.CCT;
+using System;
+
+namespace Gordon360.Models.ViewModels.RecIM
+{
+    public class MatchTeamViewModel
+    {
+        public int ID { get; set; }
+        public int TeamID { get; set; }
+        public int MatchID { get; set; }
+        public int StatusID { get; set; }
+        public int Score { get; set; }
+        public int? Sportsmanship { get; set; }
+
+        public static implicit operator MatchTeamViewModel(MatchTeam m)
+        {
+            var vm = new MatchTeamViewModel
```

Comment:
```suggestion
            var matchTeamVM = new MatchTeamViewModel
```
I would call this matchTeamVM, but that is kind of preference over requirement. You decide :)